supply-chain-security/supply-chain-security-paper/sscsp.md

```diff
@@ -1039,7 +1025,7 @@ It is critical to note, however, that organizations are entirely responsible for
      https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html
 
 [^6]:
```

Comment:
Looking for feedback; I started to go through some of the footnotes and fixing the links for the Links check, but is this the approach we want to take for the `md`?  If yes, I can continue to apply this to get the Links check to succeed.

In this specific example, it [failed](https://github.com/cncf/tag-security/runs/3361453909#step:3:40) previous to da401ee with:

```bash
[✖] [https://github.com/cncf/tag-security/tree/main/supply-chain-security/compromises](https://github.com/cncf/tag-security/tree/main/supply-chain-security/compromises) → Status: 400 [Error: ENOENT: no such file or directory, access '/usr/src/app/supply-chain-security/supply-chain-security-paper/[https:/github.com/cncf/tag-security/tree/main/supply-chain-security/compromises](https:/github.com/cncf/tag-security/tree/main/supply-chain-security/compromises)'] {
```

but now [it succeeds](https://github.com/cncf/tag-security/runs/3361637631#step:3:36):

```bash
[✓] https://github.com/cncf/tag-security/tree/main/supply-chain-security/compromises → Status: 200
```

Reply:
Ok I'll wrap up the adjustments to get this to pass then.

Reply:
@lumjjb and @TheFoxAtWork this should be all set.  Two notes in new comments for consideration.